build.gradle

```diff
@@ -2,19 +2,33 @@ buildscript {
     repositories {
         mavenLocal()
         mavenCentral()
+        jcenter()
```

Comment:
there's no point in having both mavenCentral and jcenter, please remove mavenCentral